dist/gmusic.js

```diff
@@ -207,6 +257,15 @@ proto.playback = {
       return GMusic.Playback.NO_SHUFFLE;
     }
   },
+  setShuffle: function (mode) {
+    assert(GMusic.Playback.SHUFFLE_MODES.indexOf(mode) !== -1,
+      'Expected shuffle mode "' + mode + '" to be inside ' +
+      JSON.stringify(GMusic.Playback.SHUFFLE_MODES) + ' but it wasn\'t'
+    );
```

Comment:
For consistency with the other JS, this ending parenthesis should be on the same line as the string:

```js
JSON.stringify(GMusic.Playback.SHUFFLE_MODES) + ' but it wasn\'t');
```